src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -137,7 +137,19 @@ internal Conversions Conversions
         /// The key is a symbol for methods or parameters, and syntax for attributes.
         /// The data is collected during testing only.
```

Comment:
>The data is collected during testing only. [](start = 12, length = 42)

Is this still accurate? Otherwise holding all this information indefinitely is likely to cause perf issues.

Reply:
> Is this still accurate?

Yes, `NullableAnalysisData` is only in tests.

---
In reply to: [595299058](https://github.com/dotnet/roslyn/pull/51851#discussion_r595299058) [](ancestors = 595299058)